composer.json

```diff
@@ -21,8 +21,8 @@
         "php-http/promise": "^1.0"
     },
     "require-dev": {
-        "phpspec/phpspec": "^2.4",
-        "henrikbjorn/phpspec-code-coverage" : "^1.0"
+        "phpspec/phpspec": "^4.0|^5.0|^6.0",
```

Comment:
phpspec 4 can be dropped?

Reply:
@GrahamCampbell In the current moment httplug support php version 7.0, but phpspec 5.0 support only 7.1, and I added phpspec 4.0 for php 7.0

Reply:
Oh, for some reason I though this package was PHP 7.1+.

In which case, we should change it to `^4.2` probably?

Reply:
@GrahamCampbell Why not ^4.3?

Reply:
for testing tools, i am happy with the most strict restrictions that still allow to build, if ^4.3 works thats better than ^4.2. we don't give anybody more flexibility if we allow 4.2